ZEP/instructions/zep0000.md

```diff
@@ -133,7 +146,21 @@ Process ZEPs may also have a status of `Active` if they are never meant to be co
 
 ### How does a ZEP becomes accepted?
 
-A ZEP is `Accepted` by the consensus of all interested contributors. We need a concrete way to tell whether consensus has been reached. When you think ZEP is ready to accept, create a new discussion in [zarr-python](https://github.com/zarr-developers/zarr-python/discussions) using ‘General’ as the category with a subject like:
+A ZEP is `Accepted` by the consensus of all interested contributors. We need a concrete way to tell whether the agreement has been reached. 
```

Comment:
For those following along, https://github.com/zarr-developers/governance/pull/17 is relevant to the discussion of "interested contributors"